src/main/java/mate/academy/service/UserService.java

```diff
@@ -15,6 +16,8 @@ public class UserService {
      * Return <code>null</code> if there is no suitable user
      */
     public User findByEmail(String email) {
-        return null;
+        return Arrays.stream(users)
```

Comment:
Just wanted to say that it is cool that you are familiar with streams, keep up the good work! :fire: